test/Extensions/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -55,7 +56,7 @@ public EHPurgeLogicTests()
         }
 
         [Fact, TestCategory("BVT")]
-        public async Task EventhubQueueCache_WontPurge_WhenUnderPressure()
+        public async Task EventHubQueueCache_WontPurge_WhenUnderPressure()
```

Comment:
The names weren't changed other than to fix the capitalization of "EventHub" in them.